### ISSUE TITLE
Feature recommand.

### ISSUE THREAD

**Issue by PogusTheWhisper** [PlatinShadow/ZuneDiscordRPC#6]
You can show the distance of the currently playing song, just like spotify?

**Comment by PogusTheWhisper** [comment]
![image](https://github.com/PlatinShadow/ZuneDiscordRPC/assets/50095486/fe192f57-a4c1-4995-9f69-b5650e02b4b3)

**Comment by gigalasr** [comment]
Yes! Something similar is actually implemented in the new version. The code is already in the repo, but i haven't had any time yet to build and release it
